app/models/card.rb

```diff
@@ -1,6 +1,7 @@
+# encoding: utf-8
 class CompareValidator < ActiveModel::Validator
   def validate(record)
-    if record.original_text.downcase  == record.translated_text.downcase
+    if record.original_text.mb_chars.downcase  == record.translated_text.mb_chars.downcase
```

Comment:
Line is too long. [90/80]<br>Unnecessary spacing detected.<br>Operator `==` should be surrounded with a single space.